_layouts/base.html

```diff
@@ -21,7 +21,7 @@
   <meta name="twitter:creator" content="@Code4Africa">
   <meta name="twitter:description" content="Stories from across the continent about FGM. Supported by Code for Africa and ICFJ">
   <meta name="twitter:title" content="#BornPerfect">
-  <meta name="twitter:image" content="https://bornperfect.africa/images/background/Header-banner.jpg">
+  <meta name="twitter:image" content="https://bornperfect.africa/img/background/Header-banner.jpg">
```

Comment:
Let's use either relative urls or Liquid template tags e.g `{{ site.url }}` to make it easier to re-use code.